pombola/core/views.py

```diff
@@ -482,6 +482,7 @@ def position(request, pt_slug, ok_slug=None, o_slug=None):
 
     context = {
         'object':     title,
+        'ok_slug':    ok_slug,
```

Comment:
(The `OrganisationKind` object is already being fetched in this function, as you'll probably have seen, so there'd be no additional query.)